### DISCUSSION TITLE
Add note via Capture

### DISCUSSION THREAD

**Discussion by pcmhatre** [chhoumann/quickadd#743] [General]
New to Obsidian and to QuickAdd. I've created a Capture DailyNote template and assigned a global shortcut. I've set it to add at the top of the 'Notes' section. The first time works as expected. But when used a second time in a day, it enters it at the beginning of the first bullet instead of as the first bullet in the second with the previous update as the second bullet.

I'm using `- **{{time}}** - {{value}}.` in the Format section. What am I missing?

**Comment by htmljenn** [comment]
I'm not 100% sure what you mean, but these are the two things that got me the first couple times using QuickAdd:

Did you click the "Insert at end of section" toggle? This will put the next bullet underneath the previous one. 

Also, in your format template for the bullet, don't forget to include a newline at the end. This will ensure that there is a newline between each bullet point.

**Comment by chhoumann** [comment]
Very late reply - sorry about that. htmljenn has it exactly right. Capture inserts your format text verbatim, so without a trailing newline the second capture of the day glues onto the end of the first bullet. End the format with `\n`:

```
- **{{time}}** - {{value}}.\n
```

QuickAdd expands the `\n` into a real line break, so each capture lands on its own line. And if you would rather have new entries appear below the previous ones instead of on top, enable "Insert at end of section". More detail here: https://quickadd.obsidian.guide/docs/Choices/CaptureChoice#insert-after
